Edgar-Alandete/challenges/cv-fullstack/client/src/App.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import './App.css';
+import Header from './components/Header';
+
+function App() {
+  return (
+    <Header />
+  );
+}
+
+App.propTypes = {
+
+};
+
+const mapStateToProps = ({ curriculum }) => ({ curriculum });
```

Comment:
You are not using the curriculum data from connection for this component, it is not necessary to connect here